src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -141,9 +144,9 @@ def get(self, request: Request, group) -> Response:
             group_id=group.id,
             count_over_threshold=len(
                 [
-                    result["stacktrace_similarity"]  # type: ignore
+                    result["stacktrace_distance"]  # type: ignore
```

Comment:
In general, we should not have any `# type:ignore`. Please ping me on a follow-up PR to remove these instances.